tezos/context/src/serialize/in_memory.rs

```diff
@@ -321,9 +329,13 @@ fn serialize_inode(
                     continue;
                 }
 
-                let inode_id = pointer.inode_id();
+                let hash_id = storage
```

Comment:
This assignment must be moved to line `321`, before the `if pointer.is_commited()`.
Or this will cause the garbage collector to collect/drop some `HashId` that are still in used, which will result in a different context hash.

This commit fix it, but it was not merged yet to `develop`:
[`02ee016` (#933)](https://github.com/tezedge/tezedge/pull/933/commits/02ee016943a5dd3a0a9b15863eb0e758e436ecee)